base-template/contributors-guide.md

```diff
@@ -1,51 +1,34 @@
-# Contributors' Guide
+# Base Template Contributors' Guide
```

Comment:
Can you remove the changes from this file from your PR? Otherwise, we'll end up with conflicts with https://github.com/thegooddocsproject/incubator/pull/8

Reply:
Hi Lana, I had edited it a couple of times, so reverting didn't seem like a good option, instead I copy-pasted the source code off your file. That way it shouldn't conflict now, right? Thanks.

Reply:
Haha, well, I guess we'll find out! :wink:  I want a few more reviews on my PR before I merge it, and we can just handle the conflicts as they arise.